src/common/KernelBase.cpp

```diff
@@ -548,6 +564,9 @@ void KernelBase::doOnceCaliMetaBegin(VariantID vid, size_t tune_idx)
     cali_set_double(Iters_Rep_attr,(double)getItsPerRep());
     cali_set_double(Kernels_Rep_attr,(double)getKernelsPerRep());
     cali_set_double(Bytes_Rep_attr,(double)getBytesPerRep());
+    cali_set_double(BytesRead_Rep_attr,(double)getBytesReadPerRep());
+    cali_set_double(BytesWritten_Rep_attr,(double)getBytesWrittenPerRep());
+    cali_set_double(BytesAtomicModifyWritten_Rep_attr,(double)getBytesAtomicModifyWrittenPerRep());
```

Comment:
```suggestion
    cali_set_double(Bytes_Read_Rep_attr,(double)getBytesReadPerRep());
    cali_set_double(Bytes_Written_Rep_attr,(double)getBytesWrittenPerRep());
    cali_set_double(Bytes_AtomicModifyWritten_Rep_attr,(double)getBytesAtomicModifyWrittenPerRep());
```
Match variable declarations in `KernelBase.hpp`